x/concentrated-liquidity/client/cli/tx.go

```diff
@@ -0,0 +1,51 @@
+package cli
+
+import (
+	flag "github.com/spf13/pflag"
+
+	"github.com/spf13/cobra"
+
+	"github.com/osmosis-labs/osmosis/v13/osmoutils/osmocli"
+	clmodel "github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/model"
+	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/types"
+)
+
+func NewTxCmd() *cobra.Command {
+	txCmd := osmocli.TxIndexCmd(types.ModuleName)
+	osmocli.AddTxCmd(txCmd, NewCreatePositionCmd)
+	osmocli.AddTxCmd(txCmd, NewWithdrawPositionCmd)
+	osmocli.AddTxCmd(txCmd, NewCreateConcentratedPoolCmd)
+	return txCmd
+}
+
+var poolIdFlagOverride = map[string]string{
+	"poolid": FlagPoolId,
+}
```

Comment:
My reasoning behind this decision was the pool-id flag exists for all tx commands. This may confuse the user if there is both a pool-id flag as well as an argument. Please let me know if you agree / disagree, am happy to change accordingly. May merge this before your reply, but can easily change in a follow up PR